builder/partials/body.hbs

```diff
@@ -0,0 +1,13 @@
+{{!-- see head.hbs for start --}}
+
+{{#if class }}
+  <body class="{{ class }}">
```

Comment:
can't you just do 
` <body class="{{ class || '' }}"> ` or something? copypasting this stuff seems like an overkill

Reply:
I hate empty attributes